admin_manual/configuration_files/external_storage/local.rst

```diff
@@ -16,7 +16,7 @@ are on Ubuntu Linux::
  sudo -u www-data chmod -R 0750 /localdir
  
 See :ref:`label-setting-strong-perms` for information on finding your HTTP 
-user with ``phpinfo``. 
+user with :ref:`label-phpinfo`. 
  
```

Comment:
I've just changed that hint. Thanks for the feedback :+1: 

Have also considered to just not backport this to 8.1 as 8.2 will be out soon and those are no real mandatory changes to the docs. But if desired i just can backport all that stuff without the local.rst changes.